### DISCUSSION TITLE
Getting git branch to show in inactive sections

### DISCUSSION THREAD

**Discussion by prizekl** [nvim-lualine/lualine.nvim#1302] [Q&A]
Hi, git branch component seems to not show up in my inactive panes. 

```
      local sections = {
        lualine_a = { 'branch', 'diff' },
        lualine_b = {},
        lualine_c = { { 'filename', path = 1 } },
        lualine_x = { 'diagnostics' },
        lualine_y = { 'progress' },
        lualine_z = { 'location' },
      }
```

I would need to define the following function to get this to work. Is there a reason why this is the case?

```
      local function get_branch()
        require('lualine.components.branch.git_branch').init()
        local branch = require('lualine.components.branch.git_branch').get_branch()
        return branch ~= nil and branch or ''
      end

```

**Comment by iton0** [comment]
From the [docs](https://github.com/nvim-lualine/lualine.nvim?tab=readme-ov-file#default-configuration). There is an inactive_sections option that you can update to include git branch. Hope this helps

**Reply by prizekl** [reply]
Thank you for your response. Sorry, I was unclear, adding the git branch to inactive_sections seems not to be working for me without using the function. For active_sections, this works perfectly.
